wrappers/wingdi.go

```diff
@@ -16,6 +16,48 @@
 
 package wrappers
 
+const (
+	CCHDEVICENAME = 32
+	CCHFORMNAME   = 32
+)
+
+type DEVMODE struct {
+	DmDeviceName       [CCHDEVICENAME]uint16

```

Comment:
We don't generally use Hungarian notation in Go, so let's strip off the `Dm` prefix here.